tests/www/views/test_views.py

```diff
@@ -599,26 +599,35 @@ def test_invalid_dates(app, admin_client, url, content):
     assert re.search(content, resp.get_data().decode())
 
 
-@pytest.mark.parametrize("enabled, dags_count", [(False, 5), (True, 5)])
```

Comment:
We weren't actually parametrizing the dag_cont, so I simplified this by just hard coding it in the test.